packet_packer.go

```diff
@@ -342,37 +342,45 @@ func (p *packetPacker) PackCoalescedPacket() (*coalescedPacket, error) {
 	return packet, nil
 }
 
-func (p *packetPacker) packCoalescedPacket(buffer *packetBuffer) (*coalescedPacket, error) {
+func (p *packetPacker) packCoalescedPacket(buffer *packetBuffer, maxPacketSize protocol.ByteCount) (*coalescedPacket, error) {
+	maxPacketSize = utils.MinByteCount(maxPacketSize, p.maxPacketSize)
+	if p.perspective == protocol.PerspectiveClient {
+		maxPacketSize = protocol.MinInitialPacketSize
```

Comment:
So as client we're always ignoring the parameter? Should we make that clearer, e.g. maxPacketSizeForServer?

Reply:
Only for *coalesced* packet sent by the client. This doesn't apply for normal packets.